ui/public/i18n/en/translation.json

```diff
@@ -341,6 +343,13 @@
     "master_users": "Master users",
     "master_users_placeholder": "Choose master users"
   },
+  "settings_queue": {
+    "title": "Queue settings",
+    "maximal_queue_lifetime": "Maximal queue lifetime",
```

Comment:
```suggestion
    "maximum_queue_lifetime": "Maximum queue lifetime",
```

Reply:
Ok then, I suggested to change because "maximal" is used very sparingly. Let's keep it